bundles/framework/divmanazer/resources/scss/visualizationform.scss

```diff
@@ -283,26 +290,44 @@ div.areaform, div.lineform {
 
     div.column1 {
         float: left;
-        width: 130px;
         height: 270px;
         padding-left: 5px;
         padding-right: 5px;
     }
 
 }
 
+div.lineform {
+    div.column1 {
+        width: 130px;
+    }
+}
+
+div.areaform div.column1,
+div.areaform div.column21 {
+    width: 115px !important;
```

Comment:
We should avoid using `!important`, because it is hard to override. Increase selector specificity instead:
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity